timescaledb/how-to-guides/multinode-timescaledb/multinode-setup.md

```diff
@@ -15,7 +15,9 @@ To set up multi-node on a self-hosted TimescaleDB instance, you need:
 The access and data nodes must begin as individual TimescaleDB instances.
 They should be hosts with a running PostgreSQL server and a loaded TimescaleDB
 extension. For more information about installing self-hosted TimescaleDB
-instances, see the [installation instructions][install].
+instances, see the [installation instructions][install]. Additionally, [high 
+availability with multi-node][multi-node-ha] can be configured to increase 
+redundancy and resilience.
```

Comment:
```suggestion
instances, see the [installation instructions][install]. Additionally, you
can configure [high availability with multi-node][multi-node-ha] to 
increase redundancy and resilience.
```

Small reword to make it more active, and also get the link on one line.

Reply:
I love working with people that take my bad writing and change it into somethings awesome.